pyproject.toml

```diff
@@ -28,6 +28,10 @@ include = ["strawberry/py.typed"]
 "Sponsor on GitHub" = "https://github.com/sponsors/strawberry-graphql"
 "Sponsor on Open Collective" = "https://opencollective.com/strawberry-graphql"
 
+[tool.poetry.group.dev.dependencies]
```

Comment:
not sure we want these deps in a distinct group

Reply:
Ideally all dev deps should go there, poetry deprecated dev dependencies

Reply:
I wanted to move everything there though I forgot.

Reply:
we can move them in another PR 😊